### ISSUE TITLE
Fix skipped EKS functional tests

### ISSUE THREAD

**Issue by gurevichdmitry** [elastic/cloudbeat#500]
**Motivation**

During implementation of EKS functional tests several test cases were skipped.
The skip reason was due missing findings. After investigation it was found that fleet-managed agent reveals eks findings as well and the problem is in Cloudbeat eks deployment file. This file will be updated as a part of [this task](https://github.com/elastic/security-team/issues/5107).
After fixing cloudbeat eks deployment file skipped test cases will be retested.

**Definition of done**
Retest the following rules
- 4.2.8
- 5.4.3
- 5.4.5

What needs to be completed at the end of this task
- []

**Out of scope**

What should not include in this task

**Related tasks/epics**

**Comment by oren-zohar** [comment]
@gurevichdmitry is this still relevant?

**Comment by gurevichdmitry** [comment]
@oren-zohar Yes, these test cases mentioned in the task are still relevant and need to be retested.
